src/components/Layout.jsx

```diff
@@ -5,64 +5,87 @@ import Footer from 'components/Footer'
 
 import 'scss/main.scss'
 
-export default ({ children, title, description, location }) => (
-  <div className="wrapper">
-    <Helmet>
-      <title>{title || 'Make Your Company Data Driven'}</title>
-      <meta name="title" content={title} />
-      <meta name="description" content={description} />
-      <meta
-        name="keywords"
-        content="Query, SQL, Visualize, Data, Insights, Dashboard, Alert, Alerts, web based sql client, integration, integrations, Amazong Redshift, BigQuery, PotsgreSQL, MySQL, TreasureData, MS SQL Server, MongoDB, ElasticSearch, Hive, Google Spreadsheets, Impala, InfluxDB, Presto, ScyllaDB, Cassandra, Amazon DynamoDB, Python, Vertica, Graphite, Stitch, URL"
-      />
-      <link rel="icon" href="/assets/images/favicon.ico" />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="32x32"
-        href="/assets/images/favicon-32x32.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="96x96"
-        href="/assets/images/favicon-96x96.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="16x16"
-        href="/assets/images/favicon-16x16.png"
-      />
+function reportUrlChange() {
+  // should match https://git.io/fjzPu
+  const data = {
+    type: 'iframe_url',
+    message: window.location.href,
+  }
+  window.parent.postMessage(data, '*')
+}
 
-      <meta name="format-detection" content="telephone=no" />
+export default class extends React.Component {
+  componentDidMount() {
+    window.addEventListener('hashchange', reportUrlChange)
```

Comment:
Why not just use `useEffect` hook instead of rewriting everything to class-based component? Like:
```
useEffect(() => {
  window.addEventListener('hashchange', reportUrlChange);
  reportUrlChange();
  return () => window.removeEventListener('hashchange', reportUrlChange);
}, []);
```
BTW - you're missing semicolons everywhere - eslint should show warnings for this.

Reply:
Regarding semicolons - sorry, I missed that it's website, not app. I take my words back.